cfe_internal/enterprise/CFE_hub_specific.cf

```diff
@@ -342,7 +342,7 @@ bundle agent cfe_internal_truncate_events
 {
   commands:
     Hr05.Min00_05::
-      "$(sys.workdir)/httpd/php/bin/php $(cfe_internal_hub_vars.docroot)/index.php cli_tasks truncate_events"
+      "$(sys.workdir)/httpd/php/bin/php $(cfe_internal_hub_vars.docroot)/public/index.php cli_tasks truncate_events"
```

Comment:
Won't this break bookmarks to links inside Mission Portal?

I am thinking things like URLs with base64 encoded sql queries, saved reports, links to individual host pages.

Reply:
URLs won't be changed, this is CLI command that requires path to index.php.

Reply:
Ok, thanks.